app/notification_push_test.go

```diff
@@ -1296,6 +1296,7 @@ func TestAllPushNotifications(t *testing.T) {
 	defer pushServer.Close()
 
 	th.App.UpdateConfig(func(cfg *model.Config) {
+		*cfg.EmailSettings.PushNotificationContents = model.GENERIC_NOTIFICATION
```

Comment:
@hmhealey what are we changing here regarding email notifications? This ticket is for push notifications

Reply:
Yeah, that's for mobile push notifications